web/packages/teleterm/src/ui/services/workspacesService/documentsService/connectToApp.ts

```diff
@@ -20,13 +20,18 @@ import { routing } from 'teleterm/ui/uri';
 import { IAppContext } from 'teleterm/ui/types';
 
 import { App } from 'teleterm/services/tshd/types';
+import { getWebAppLaunchUrl, isWebApp } from 'teleterm/ui/services/clusters';
 
 import { DocumentOrigin } from './types';
 
 export async function connectToApp(
   ctx: IAppContext,
   target: App,
-  telemetry: { origin: DocumentOrigin }
+  telemetry: { origin: DocumentOrigin },
+  options?: {
+    /** If the app is a web app, it will be launched in a browser. */
+    launchInBrowser: boolean;
```

Comment:
Just an idea, idk if it's better in any way. But when looking at the callsite in actions.tsx alone, someone might think that we launch all apps in the browser, which is not true.

```suggestion
    launchInBrowserIfWebApp: boolean;
```

Reply:
I was considering something like that, changed.